Makefile

```diff
@@ -14,7 +14,7 @@ test: build-node
 .PHONY: test-without-build
 test-without-build:
 	npm run flow
-	npm run lint
+	#npm run lint
```

Comment:
this seems suspicious

Reply:
Yeah, I should take out the commented out line.  The reason `npm run lint` is being taken out is that we have reconfigured Travis to lint on LTS versions of node.